### PR TITLE
Remember token locally with `gr.load()`

### PR DESCRIPTION
**Pull request by abidlabs**
This PR now adds a "Remember me" checkbox to `gr.load()` when `accept_token=True`. The token is stored in local storage using the new `gr.BrowserState` demo. Here's a video of it in action (you'll notice some other UI updates as well):

https://github.com/user-attachments/assets/0a26d644-6721-45aa-ac70-67754108d62f

Includes a test demo: `demo/load_model_with_token/run.py` but this should also work with any of our `gr.load` integrations as well. cc @AK391 @yvrjsharma

### CODE REVIEW

File: gradio/external.py [hunk 1]
```diff
@@ -89,11 +89,41 @@ def load(
...
+                if remember_token_value and token_value:
+                    return token_value, gr.Checkbox(
+                        label="Remember me on this device (saved!)", value=True
+                    )
+                else:
+                    return "", gr.Checkbox(label="Remember me on this device")
+
+            @gr.on(demo.load, inputs=[browser_state], outputs=[textbox, remember_token])
+            def load_token(token_value):
+                print("load_token", token_value)
```
Comment (aliabid94) [inline gradio/external.py:122]: remove print

Comment (abidlabs) [reply gradio/external.py]: 👍

**Review by aliabid94 — APPROVED** [review]
lgtm!

**Comment by abidlabs** [comment]
Thanks @aliabid94!

**Pull request merged**